static/css/legacy-borrowTable-adminUser.less

```diff
@@ -5,30 +5,77 @@
  */
 @import (less) "less/font-families.less";
 
-body#admin,
-body#user {
-  /* stylelint-disable selector-max-specificity */
-  #borrowTable {
-    margin: 30px 0 25px;
-    table {
-      width: 100%;
-      font-family: @lucida_sans_serif-7;
-      th {
-        border-bottom: 1px solid @beige-three;
-        vertical-align: bottom;
-        padding-bottom: 10px;
-        font-family: @lucida_sans_serif-1;
-        color: @brown;
-        font-weight: 600;
-        font-size: .6875em;
-        text-transform: uppercase;
-        white-space: nowrap;
-        text-align: center;
-        &.titles {
-          text-align: left;
-        }
+/* stylelint-disable selector-max-specificity */
+#borrowTable {
+  margin: 30px 0 25px;
+  table {
+    width: 100%;
+    font-family: @lucida_sans_serif-7;
+    th {
+      border-bottom: 1px solid @beige-three;
+      vertical-align: bottom;
+      padding-bottom: 10px;
+      font-family: @lucida_sans_serif-1;
+      color: @brown;
+      font-weight: 600;
+      font-size: .6875em;
+      text-transform: uppercase;
+      white-space: nowrap;
+      text-align: center;
+      &.titles {
+        text-align: left;
       }
     }
   }
-  /* stylelint-enable selector-max-specificity */
+}
+tr {
+  &.row-available {
+    background: @admin-row-background !important;
+    .status-value {
+      color: @dark-green-two;
```

Comment:
Let's use @dark-green here rather than create a new variable. The existing variable already has a confusing name :)

We can always change those colors later.